### PR TITLE
[MFTF] create AssertLinkActionGroup

### LINKED ISSUE #30858: [Issue] [MFTF] create AssertLinkActionGroup
This issue is automatically created based on existing pull request: magento/magento2#30837: [MFTF] create AssertLinkActionGroup

---------
This PR provides `AssertLinkActionGroup`

### PR DESCRIPTION
**Pull request by Usik2203**
This PR provides `AssertLinkActionGroup` 

### Resolved issues:
1. [x] resolves magento/magento2#30858: [MFTF] create AssertLinkActionGroup

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run  Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by sidolov** [comment]
@magento create issue

**Comment by Usik2203** [comment]
@magento run  Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
Hi @rogyar 
 Could I kindly ask you take a look on this one ?  ⬆️ 
Thank you.

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertLinkActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertLinkActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertLinkActionGroup.xml:11]: 👍

**Review by rogyar — APPROVED** [review]
The failing tests are not related to the current changeset

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8457](https://jira.corp.magento.com/browse/ENGCOM-8457) has been created to process this Pull Request

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**